src/tools/zimDump.cpp

```diff
@@ -289,7 +289,6 @@ void ZimDumper::listArticle(const zim::Article& article, bool extra)
         case zim::zimcompDefault: std::cout << "default"; break;
         case zim::zimcompNone:    std::cout << "none"; break;
         case zim::zimcompZip:     std::cout << "zip"; break;
-        case zim::zimcompBzip2:   std::cout << "bzip2"; break;
```

Comment:
Same here. Even if we totally drop the bzip2 support, if we found a zim with bzip2 compression, we know this is a bzip2 compression. It is better than `unknown`.

We may change the displayed text to `"bzip2 (not supported)"`